frontend/src/components/layouts/FixedHeightLayout.js

```diff
@@ -19,13 +22,44 @@ const useStyles = makeStyles({
   },
 });
 
-export default function FixedHeightLayout({ title, children }) {
+export default function FixedHeightLayout({ 
+  title, 
+  children,
+  message,
+  messageType
+}) {
   const classes = useStyles();
-
+  const [initialMessageType, setInitialMessageType] = React.useState(null);
+  const [alertOpen, setAlertOpen] = React.useState(true);
+  const [initialMessage, setInitialMessage] = React.useState("");
+  useEffect(() => {
+    const params = getParams(window.location.href);
+    if (params.message) setInitialMessage(decodeURI(params.message));
```

Comment:
Nit: do you mind adding braces for one-liners? I find it slightly easier to debug. This is approaching bikeshedding IMO so definitely welcome a push back here 🙏 
```suggestion
    if (params.message) {
        setInitialMessage(decodeURI(params.message));
    }
```

Reply:
I agree that this makes it easier to read so I'm fine with doing this in the future. As long as it offers real benefits such as readability I always welcome suggestions